packages/tasks/src/model-libraries.ts

```diff
@@ -280,7 +280,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/NVIDIA/NeMo",
 		snippets: snippets.nemo,
 		filter: true,
-		countDownloads: { wildcard: { path: "*.nemo" } },
+		countDownloads: { wildcard: { path: "*.nemo", "model_config.yaml" } },
```

Comment:
i think this is not valid JS or JSON

Reply:
Sorry can you elaborate? I see `speechbrain` is [tracking](https://github.com/huggingface/huggingface.js/blob/426228b630f90674553dd2b7738e01051001e8e3/packages/tasks/src/model-libraries.ts#L389) `hyperparams.yaml`